Chaincase.UI/Components/TransactionListItem.razor

```diff
@@ -0,0 +1,39 @@
+﻿@using Chaincase.Common.Contracts
+@using Chaincase.UI.Services
+
+@inherits ReactiveComponentBase<TransactionViewModel>
+
+<ion-item>
+    @if(!ViewModel.Confirmed)
+    {
+        <ion-icon slot="start"
+                  style="color: grey;"
```

Comment:
Should this be in a style sheet?

Reply:
ya probably!

Reply:
I'm OK with component-specific styles if instead of constant colors you can set them to a variable but that's getting pretty weedy isn't it. Doing that you'd be able to control colors / fonts / styles from a global stylesheet as rules and then tweak their application in the component

Reply:
wait ... after further thought what I'm describing is just making use of classes